supply/lib/supply/uploader.rb

```diff
@@ -181,7 +181,7 @@ def check_superseded_tracks(apk_version_codes)
       max_apk_version_code = apk_version_codes.max
       max_tracks_version_code = nil
 
-      tracks = ["production", "rollout", "beta", "alpha"]
+      tracks = ["production", "rollout", "beta", "alpha", "internal"]
```

Comment:
why not `%w(production beta alpha internal rollout)`?

Reply:
Either way is fine, we can leave it as is